python/setup.py

```diff
@@ -1,48 +1,35 @@
-from setuptools import setup
-from setuptools.command.install import install
-import os
-import sys
-import pypandoc
+# SPDX-License-Identifier: Apache-2.0
 
-class ManPageGenerator(install):
```

Comment:
We can't do this; Debian and Fedora both require a man page to be installed for every installed executable. I can add it back in for you, but we can't remove it. I'll change the authorship in them.

Reply:
Feel free to modify. I only included the setup.py so it can be built quickly. Never expected it to stay the unchanged.